app/_posts/blog/2025-03-17-security-policy-update.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: blog
+title: How Yeoman is Enhancing Security Through Policy Updates
+---
+
+This is just the start. We're also working on a **deeper security initiative** to strengthen Yeoman for the long haul. If you're curious or want to contribute, check out the discussion [here](https://github.com/yeoman/.github/issues/1).
+
+---
+We're not just dusting off old code. As part of our **[Maintenance Reboot Initiative](https://yeoman.io/blog/maintenance-reboot)**, we're rebuilding Yeoman for the future—stronger, faster, and definitely more secure. 
+
+That’s why we’ve rolled out an updated **[Security Policy](https://github.com/yeoman/.github/blob/main/SECURITY.md)** to keep our ecosystem locked down tight. If you're into open source (and we know you are), here's how you can help keep Yeoman secure.
+
+## How to Report a Vulnerability (Without Blowing Up the Internet)
+
+- **Found something shady?**  
+  - **Step 1:** **Do NOT open a public issue!** 
+  - **Step 2:** Report it privately through [GitHub Security Advisory](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability).  
+  - **Step 3:** Can’t use GitHub? Check the [security policy](https://github.com/yeoman/.github/blob/main/SECURITY.md) for an alternative option.
+
+Why private? Because public issues can give attackers a head start, and we’re not here for that.
+
+## What Happens After?
+
+- You'll hear from us within **2–5 working days** (we're fast, but coffee breaks exist).  
```

Comment:
> 2-5 working days

(confirming this internally)

Reply:
It is for the acknowledgement https://github.com/yeoman/.github/blob/main/SECURITY.md?plain=1#L37 but maybe is not clear :thinking:

Reply:
Oh I see - "hear from us". Yup that makes sense, I think that's fine. 👍

Reply:
I just made an update https://github.com/yeoman/.github/pull/24 (until we can have an email alias in place)